lib/components/narrative/tabbed-itineraries.js

```diff
@@ -64,49 +66,57 @@ class TabbedItineraries extends Component {
         !useRealtime
       )
     return (
-      <div className='options itinerary tabbed-itineraries'>
-        <div className='tab-row'>
-          {itineraries.map((itinerary, index) => {
-            return (
-              <TabButton
-                currency={currency}
-                defaultFareKey={defaultFareKey}
-                index={index}
-                isActive={index === activeItinerary}
-                itinerary={itinerary}
-                key={index}
-                onClick={setActiveItinerary}
-              />
-            )
-          })}
-        </div>
+      <>
+        {error && itineraries && <div className='error-message'>
+          <div className='header'>
+            <i className='fa fa-exclamation-circle' /> Warning
+          </div>
+          <div className='message'>{error.msg}</div>
+        </div>}
+        <div className='options itinerary tabbed-itineraries'>
+          <div className='tab-row'>
+            {itineraries.map((itinerary, index) => {
+              return (
+                <TabButton
+                  currency={currency}
+                  defaultFareKey={defaultFareKey}
+                  index={index}
+                  isActive={index === activeItinerary}
+                  itinerary={itinerary}
+                  key={index}
+                  onClick={setActiveItinerary}
+                />
+              )
+            })}
+          </div>
 
-        {/* <RealtimeAnnotation
+          {/* <RealtimeAnnotation
           realtimeEffects={realtimeEffects}
           toggleRealtime={this._toggleRealtimeItineraryClick}
           useRealtime={useRealtime} />
         */}
 
-        {/* Show active itin if itineraries exist and active itin is defined. */}
-        {(itineraries.length > 0 && activeItinerary >= 0)
-          ? (
-            <ItineraryBody
-              active
-              expanded
-              index={activeItinerary}
-              itinerary={itineraries[activeItinerary]}
-              key={activeItinerary}
-              LegIcon={LegIcon}
-              routingType='ITINERARY'
-              showRealtimeAnnotation={showRealtimeAnnotation}
-              timeFormat={timeFormat}
-              {...itineraryBodyProps}
-            />
-          )
-          : null
-        }
+          {/* Show active itin if itineraries exist and active itin is defined. */}
+          {(itineraries.length > 0 && activeItinerary >= 0)
+            ? (
```

Comment:
pretty sure this could just be `(condition) && <component>`

Reply:
although it looks like the only change here was indentation, so maybe you don't want to change this.

Reply:
yeah usually we don't refactor unless we're directly touching that line. I try to not commit auto-formatted noise but it looks like this one slipped through!